tasks/transform_conformance/babel.snap.md

```diff
@@ -742,42 +742,33 @@ Passed: 164/1083
 * enum/ts5.0-const-foldable/input.ts
 * exports/declared-types/input.ts
```

Comment:
I think Bable is wrong.

`declare` does not mean `Type`, it means that the variable has been defined in this scope.
It may be defined by the host, such as `document`, or it may be a JS global variable, such as `Array`.
However, it is a value rather than a type, so it should not be removed directly.

I will file a issue to Babel.